docs/htdocs/info/docs/tools/vep/script/vep_other.html

```diff
@@ -256,26 +256,27 @@ <h4 id="pick_options">Options</h4>
     <li style="margin-bottom:20px">
       <b>--pick</b></p>
       <p>This is the option
```

Comment:
Please also delete "This is the option", as it makes no sense without the other half of the sentence.